<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRCode</class>
 <widget class="QWidget" name="QRCode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QRCode</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>9</y>
     <width>96</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入二维码信息：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>10</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>二维码测试--Hello World！</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>生成</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>421</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
     border: 1px solid gray;
     border-radius: 5px;
     margin-top: 1ex; 
 }

 QGroupBox::title {
     color:green;
     subcontrol-origin: margin;
     subcontrol-position: top center; 
     padding: 0 3px;
 }</string>
   </property>
   <property name="title">
    <string>二维码</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>二维码图片：</string>
    </property>
   </widget>
   <widget class="QImageViewer" name="imageWidget" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <widget class="QImageViewer" name="logowidget" native="true">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>添加Logo</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QImageViewer</class>
   <extends>QWidget</extends>
   <header>qimageviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
